src/ecs_deplojo/task_definitions.py

```diff
@@ -55,6 +62,16 @@ def set_environment(self, env: typing.Dict[str, str]):
         for container in self.container_definitions:
             container["environment"] = env
 
+    def set_secrets(self, secrets: typing.Dict[str, str]):
+        """Interpolate all the secrets used in the task definition.
+
+        Secrets will be fetched from the AWS Parameter store and injected in the
```

Comment:
This can also be retrieved from AWS Secret Manager toch?

Reply:
yes looks like it: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data.html 

didn't try that though.